Sources/PT.PM.Common/Nodes/Tokens/BinaryOperator.cs

```diff
@@ -41,6 +41,10 @@ public enum BinaryOperator
 
         // JavaScript
         In,               // in
-        InstanceOf        // instanceof
+        InstanceOf,       // instanceof
+
+        //Python
+        Is,               // IS
```

Comment:
Strictly speaking `is not` and `not in` are whole binary operators on their own in Python, I don't know whether current parser will handle them correctly.

Reply:
maybe it's better to represent "is not" and "not in" like  negation of "is" and "in" expressions? just for greater versatility